### PR TITLE
Add Elementz in Showcase

### PR DESCRIPTION
**Pull request by el1s7**
Thanks for making styleguidist <3 !
 I've used it for documenting my latest project https://github.com/elementz-ui/elementz and would like to include it in the showcase examples if possible.

### CODE REVIEW

File: site/src/pages/index.js [hunk 1]
```diff
@@ -67,16 +67,16 @@ const examples = [
 		href: 'https://react-styleguidist.js.org/examples/basic/',
 		image: '/img/example1.png',
 	},
+	{
+		title: 'Elementz UI',
+		href: 'https://elementz.style',
+		image: 'https://elementz.style/showcase.PNG'
```
Comment (sapegin) [inline site/src/pages/index.js:73]: Please update an existing image and make it the same size and style as the other images (it should be an actual screenshot). Don't forget to optimize an image too.

Also, no need to change the order.

**Review by sapegin — CHANGES_REQUESTED** [review]
Thanks!

File: site/src/pages/index.js (hunk 1 above)
Comment (el1s7) [reply site/src/pages/index.js]: I have resized it to 590x473. I'm not sure what you mean by style and order, it is an actual screenshot, if you mean to use the Light theme then you can add this instead https://elementz.style/showcase_white.png

Comment (sapegin) [reply site/src/pages/index.js]: Yup, light theme would better match the other screenshots. And it's also the default.

By order I mean the order or examples on the page.

Comment (el1s7) [reply site/src/pages/index.js]: Feel free to edit it

Comment (el1s7) [reply site/src/pages/index.js]: @sapegin

**Pull request closed without merging**